File: src/main/java/com/github/duc010298/web_api/entity/socket/AppMessage.java
package com.github.duc010298.web_api.entity.socket;

public class AppMessage {
	private String command;
    private String imei;
    private Object content;

    public String getCommand() {
        return command;
    }

    public void setCommand(String command) {
        this.command = command;
    }

    public String getImei() {
        return imei;
    }

    public void setImei(String imei) {
        this.imei = imei;
    }

    public Object getContent() {
        return content;
    }

    public void setContent(Object content) {
        this.content = content;
    }
}

File: winform_client/winform_client/Entity/AppMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace winform_client.Entity
{
    class AppMessage
    {
        public string command { get; set; }
        public string imei { get; set; }
        public Object content { get; set; }
    }
}
